2021-05-06 v. 1.0.7: added in-app update feature

vipozdn opened the pull request:
1.Added:
- play core library dependency in app/build.gradle
- flexible in-app update feature in MainActivity

## Checklist

__Common__

- [x] I am ran the app before creating PR
- [x] I am ran all tests before creating PR

__UI__

- [x] I am ran the app for visual checks.

__Logic__

- [x] I am tested basic app functionality.

## Changes

Describe all changes here:

- First;
- Second;
- Third;
- ...

## Comments

Describe all additional information here.

vipozdn committed:
2021-05-06 v. 1.0.7: added in-app update feature
1.Added:
- play core library dependency in app/build.gradle
- flexible in-app update feature in MainActivity

fartem commented on app/src/main/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java:
```diff
@@ -27,6 +38,9 @@
     public static final String URI_FROM_EXTERNAL_APP
             = "com.smlnskgmail.jaman.hashcheckerlite.URI_FROM_EXTERNAL_APP";
 
+    private static final int MY_REQUEST_CODE = 1;
```
Please rename `MY_REQUEST_CODE` to `REQUEST_APP_UPDATE`.

fartem commented on app/src/main/java/com/smlnskgmail/jaman/hashcheckerlite/MainActivity.java:
```diff
@@ -163,4 +194,49 @@ public void onBackPressed() {
+        Snackbar snackbar =
+                Snackbar.make(
+                        findViewById(android.R.id.content).getRootView(),
+                        "An update has just been downloaded.",
```
You need to move all strings in UI to `strings.xml` file in resources. I'm translating this sentences to other languages.

vipozdn committed:
05-07-2021 v. 1.0.7 added strings to resources

1. Added:
- strings from UI to strings.xml

2. Updated:
- MY_REQUEST_CODE to REQUEST_APP_UPDATE in MainActivity

fartem committed:
Merge branch 'master' into feature

fartem commented:
@vipozdn I'm added you to Contributors list in README.md.

vipozdn commented:
> 
> 
> @vipozdn I'm added you to Contributors list in README.md.

That's cool! Tnx!
